templates/app/aspnet-core/src/MyCompanyName.MyProjectName.Blazor.Server.Tiered/Pages/Index.razor

```diff
@@ -2,6 +2,9 @@
 @using Volo.Abp.MultiTenancy
 @inherits MyProjectNameComponentBase
 @inject AuthenticationStateProvider AuthenticationStateProvider
+
+<PageTitle>Index</PageTitle>
```

Comment:
Should consider localization

Reply:
I removed PageTitle from Index.razor pages (https://github.com/abpframework/abp/commit/09be568ee24f5189fb3b4bb2b972f1cf31447d2f) and updated logic that renders app name when it's not presented (https://github.com/abpframework/abp/commit/5478f6d9de58d2f835ce170383e75e9951167300)